### DISCUSSION TITLE
[Novo App Componentes] - Sheet Input Text

### DISCUSSION THREAD

**Discussion by LucasAlbuquerque0** [Telefonica/mistica-design#1587] [Proposals]
### Goal

There was a need to input data and information into the component.

Text Field component (Nestaded Instance allowing the choice of one line or multiple lines.
SheetAction.

### Context

We need the component to use in my application requests. With the addition of this component we can include new functionalities such as capturing quick information, in addition to applying coupons.

### Impact

Reduction of new screens needed to capture text and data, keeping them within the same screen using just one sheet.

### Discussion Questions

It is possible to add new data capture formats within the field, such as rating and text.

### Do you have a visual proposal?

https://www.figma.com/file/fVf7qaHquX7rOdDNQAmn2w/Icon-Consumo?type=design&node-id=1%3A86&mode=design&t=PAQeEqSjZMlsRrr4-1

**Comment by yceballost** [comment]
I think the link to visual proposal is wrong @LucasAlbuquerque0 🤔

**Reply by LucasAlbuquerque0** [reply]
Sorry, i ended up changing the links. Follow the correct link
https://www.figma.com/file/3FWICps6yc9do1FVBRxxgN/Bottom-Sheet-Evolu%C3%A7%C3%A3o?type=design&node-id=432%3A6211&mode=design&t=DGXLSed7uv5xYYBX-1

**Reply by yceballost** [reply]
What happens with the keyboard? I think this case can generate a lot of interactions problems

Can you provide real examples of this pattern?

We are going to ask to dev team if its possible to include the keyboard inside the sheet and move all the content with the keyboard (like fixedButtonFooterLayout)

**Reply by LucasAlbuquerque0** [reply]
Of course, I updated it with behavior examples.
And I put some references to how other applications currently use this behavior.

My main reference is 01.
What do you think?

https://www.figma.com/file/3FWICps6yc9do1FVBRxxgN/Bottom-Sheet-Evolu%C3%A7%C3%A3o?type=design&node-id=534-2744&mode=design&t=sdVZsa7N8DnPVewY-4
